packages/nextjs/app/builders/0x48Bf488D00FE4C83e803688ffE5532EB83E9a0ff/page.tsx

```diff
@@ -0,0 +1,121 @@
+"use client";
```

Comment:
We don't need this restriction since your page doesn't use any client-only mechanism.
Please remove this line so we can improve the app's performance by pre-rendering the page during the compile process.